app/realm/actions/channel.js

```diff
@@ -638,6 +639,12 @@ export function markChannelAsFavorite(channeId, favorite = true) {
     };
 }
 
+export function markChannelsViewedAndRead(channelId, previousChannelId, markOnServer = true) {
+    return () => {
+        reduxStore.dispatch(markChannelViewedAndRead(channelId, previousChannelId, markOnServer));
```

Comment:
Why is this needed? Aren’t we migrating?

Reply:
Unless I read this wrong, this is updating the server side itself. Should I instead create an Action in the channel (realm/action/channel) that dispatches to client4 to update?